package.json

```diff
@@ -32,7 +32,13 @@
     "jest:test": "jest --coverage",
     "plugin:hello-world": "tsx tests/__mocks__/hello-world-plugin.ts",
     "setup-kv": "bun --env-file=.dev.vars scripts/setup-kv-namespace.ts",
-    "setup": "tsx ./scripts/setup.ts"
+    "setup": "tsx ./scripts/setup.ts",
+		"start:azure": "run-p proxy build:watch start",
```

Comment:
is the formatting correct or is it a render bug in my GitHub iOS client

Reply:
I think the space is a tab most likely, I will fix that.

Reply:
Do you think run-s might make more sense, to run in series instead of parallel? I suggest that because build generally should complete before starting.

Reply:
It's true but since we are watching the code to rebuild on save, I think it makes more sense to run in parallel otherwise saving won't reload the code.